dynamo/src/main/java/no/bibsys/db/structures/SimpleEntry.java

```diff
@@ -37,5 +41,7 @@ public void setPreferredLabels(final List<LanguageString> labels) {
         this.preferredLabels = labels;
     }
 
-
+    public String asJson() throws JsonProcessingException {
+        return ObjectMapperHelper.getObjectMapper().writeValueAsString(this);
```

Comment:
Maybe should be declared in the interface? Since it is declared both in IdOnlyEntry and SImpleEntry

Reply:
Also, SImpleEntry is used only in tests. It was quite stupid of me to add them to the production code, maybe more appropriate to move this class  along with `LanguageString` to tests?